lib/pbench/server/api/resources/datasets_list.py

```diff
@@ -345,14 +336,16 @@ def keyspace(self, query: Query) -> JSONOBJECT:
         aggregate: JSONOBJECT = {
             "dataset": {c.name: None for c in Dataset.__table__._columns}
         }
-        list = query.with_entities(Metadata.key, Metadata.value).all()
-        for k, v in list:
-            # "metalog" is a top-level key in the Metadata schema, but we
-            # report it as a sub-key of "dataset".
-            if k == Metadata.METALOG:
-                self.accumulate(aggregate["dataset"], k, v)
-            elif k:
-                self.accumulate(aggregate, k, v)
+
+        Database.dump_query(query, current_app.logger)
```

Comment:
Adding the default argument here makes it clear to the reader that the function is intended to be active only for debugging (otherwise, this behavior is implicit, and the concerned reader has to go find the definition of `dump_query()` to figure that out).  (We can keep the argument's default value for compatibility with other, existing code; but it would be good from a code-as-documentation perspective to supply the value explicitly here.)